.github/workflows/code-quality.yaml

```diff
@@ -51,3 +51,13 @@ jobs:
           echo "Starting Hadolint"
           find . -name "Dockerfile" | xargs ./hadolint --config ./ci/hadolint-config.yaml
           echo "Hadolint done"
+
+      # This simply checks that the manifests and respective kustomization.yaml finishes without an error.
+      - name: Check kustomize manifest
+        id: kustomize-manifests
+        run: |
+          type kubectl || echo TODO
```

Comment:
`set -x`, to avoid having to write these echos?

Reply:
maybe in the future - I added also extra lines with `----` separators for better text separation visually

Reply:
banners; meh

![image](https://github.com/opendatahub-io/notebooks/assets/442720/381a7b30-045e-49c5-8341-900c5dafa8bb)

;P